### DISCUSSION TITLE
How to map keyboard command to vim search?

### DISCUSSION THREAD

**Discussion by nicholaswhittle** [VSCodeVim/Vim#8103] [Q&A]
I can't find the vim command names.

I'm trying to remap CMD + f so it uses vim search instead of vscode within files

**Comment by Acumane** [comment]
If you've set `"vim.useCtrlKeys"` to `false`, you'll need to return control back over to Vim for `<C-f>`:
```
"vim.handleKeys": {
	"<C-f>": true
}
```

**Comment by nicholaswhittle** [comment]
I'm on mac, does this still apply?

Is CMD key not supported by vscodevim?
If not, is there a way I can trigger a vim command outside of `normalModeKeyBindingsNonRecursive` such as native vscode keybindings

**Reply by Acumane** [reply]
"C" in `<C-f>` is understood as CMD on Mac. Try adding the block I sent above to your `settings.json`

I know `"vim.useCtrlKeys"` suggests it's only for CTRL, but I suspect it'll work the same

**Comment by nicholaswhittle** [comment]
Okay cool, what is the command name for vim to start a new search in file (/ command)?

**Reply by Acumane** [reply]
Pressing `/` opens the Vim search prompt in your status bar when not in insert mode.

If your initial question was solved, mark the solution as Answer for other's future reference
